src/subscription.ts

```diff
@@ -472,20 +497,31 @@ export class Subscription extends Subscriber {
    *   const apiResponse = data[1];
    * });
    */
-  get(gaxOpts, callback) {
-    if (is.fn(gaxOpts)) {
-      callback = gaxOpts;
-      gaxOpts = {};
-    }
+  get(callback: CreateSubscriptionCallback): void;
+  get(gaxOpts?: CallOptions&GetCallOptions): Promise<Subscription>;
```

Comment:
I believe `CreateSubscriptionResponse` could be imported from `index.ts` and used here => Promise<CreateSubscriptionResponse>